### DISCUSSION TITLE
引用会议的小技巧

### DISCUSSION THREAD

**Discussion by CZLeader** [BITNP/BIThesis#477] [Show and tell]
以下技巧用于满足一些个性化的引文格式要求。

国标要求的引用会议论文格式为

```
贾东琴,柯平.面向数字素养的高校图书馆数字服务体系研究[C]//中国图书馆学会.中国图书馆学会年会论文集:2011年卷. 北京:国家图书馆出版社,2011:45-52.
```

需要满足个性化的引文格式，如

```
作者. 文献名. 会议名[C] 地点. 举办者. 其他信息
```

即

```
贾东琴,柯平.面向数字素养的高校图书馆数字服务体系研究. 中国图书馆学会年会论文集:2011年卷.[C]北京:国家图书馆出版社,2011:45-52.
```

格式的，可以

- 将`@inproceedings`改为`@proceedings`，
- 然后将`booktitle`的内容挪入`title`标签内，并将`booktitle`删除，即修改题名为`面向数字素养的高校图书馆数字服务体系研究. 中国图书馆学会年会论文集:2011年卷`。

**Comment by CreeperLKF** [comment]
如果只是需要去除`//`双斜线的话，可以在下面新增选项 `gbpunctin=false`
https://github.com/BITNP/BIThesis/blob/2f6cf63d054e67bc7a07948a02d2f03f5521a075/templates/undergraduate-thesis-en/main.tex#L74-L84
效果如下：
```bibtex
@inproceedings{dwarkadasCashmereVLMRemoteMemory1999,
  title = {Cashmere-{{VLM}}: {{Remote}} Memory Paging for Software Distributed Shared Memory},
  shorttitle = {Cashmere-{{VLM}}},
  booktitle = {Proceedings 13th {{International Parallel Processing Symposium}} and 10th {{Symposium}} on {{Parallel}} and {{Distributed Processing}}. {{IPPS}}/{{SPDP}} 1999},
  author = {Dwarkadas, S. and Hardavellas, N. and Kontothanassis, L. and Nikhil, R. and Stets, R.},
  year = {1999},
  month = apr,
  pages = {153--159},
  doi = {10.1109/IPPS.1999.760451},
}
```
![image](https://github.com/BITNP/BIThesis/assets/28673129/fda92b9d-eb61-4d60-9fe9-732b022f58be)

还有其他需求可以参考：
https://github.com/TheNetAdmin/zjuthesis/issues/117
https://github.com/sjtug/SJTUThesis/issues/337

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq/)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/bib-conference.html
https://github.com/BITNP/BIThesis-wiki/blob/6a76c26ca78435550e068339ba1c31ce55622c69/wiki/faq/bib-conference.md
